madgiraffestudios/src/components/About/About.js

```diff
@@ -0,0 +1,100 @@
+                    <p className="font-weight-900 mb-5">
+                        The best way to spark change is to start a conversation, and there’s no better way to start conversation than with art. For me, the best way to communicate with others is through art; it allows tough conversations to be had and to be learned. Art inspires me to make social change and inspire others to truly think.
+            </p>
+                </div>
+            </div>
+        </div>
+        <div className="container mt-3" id="aboutlower">
+
+            <div className="row about-content">
+                <div className="col-md-6" id="first">
+                    <img src={photoshoots} className="img-fluid rounded float-left" alt="Responsive image" />
+                </div>
+                <div className="col-md-6 mt-5 p-3" id="second">
+
+                    <h5 className="font-weight-bold primary-color">Photo Shoots</h5><p className="w-75 text-muted">
+                        At Madgiffare studios I am dedicated to capturing the moment as imagined, by you! The photoshoot is yours, this is your art. Look your best in a variety of shoots including, product photography, portraits, street photography, and events.</p>
+                </div>
+                <div>
+
+                </div>
+
+                <div className="row">
+                    <div className="col-md-6 mt-5 p-3" id="second">
+                        <h5 className="font-weight-bold primary-color">Graphic Design</h5><p className="w-75 text-muted">Your improvement as a business is my priority, what a business displays is extremely important to how they’re portrayed. Using custom logos, and having a strong Social media presence graphically will truly boost Company Branding and popularity.</p>
+                    </div>
+                    <div className="col-md-6" id="first">
+                        <img src={graphicdesign} alt="performance" className="img-fluid rounded float-right" />
+                    </div>
+                </div>
+
+
+                <div className="row">
+                    <div className="col-md-6" id="first">
+                        <img src={drawing} alt="performance" className="img-fluid rounded float-left" />
+                    </div>
+                    <div className="col-md-6 mt-5 p-3" id="second">
+                        <h5 className="font-weight-bold primary-color">Illustrations</h5>
+                        <p className="w-75 text-muted">Now imagine having your favorite photo of yourself, or a loved one reimagined through in an Illustration.Choose from a hand drawing,painting or digital art and let me bring your imagination to life.                                                                                            Get started today!
+                           </p>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+
+
+        <div class="footer_bg pt-5 pb-3 bg-light text-center">
```

Comment:
- [ ] Footer should be in a different component. This can be a separate issue

Reply:
hold on